aries-site/src/pages/components/textinput.js

```diff
@@ -17,21 +26,144 @@ const TextInput = () => (
     />
     <ContentSection>
       <Subsection name={title} level={1} topic={topic}>
-        <SubsectionText>
-        The TextInput component allows the user to input shorter forms
-        of data and content. Passwords and tags can also be used with
-        the TextInput component. Style can be variable, based upon
-        the use case and customer need that will elicit user
-        confidence in success.
-        </SubsectionText>
+        <SubsectionText>{page.description}</SubsectionText>
         <Example
+          designer="https://designer.grommet.io/textinput?id=HPE-design-system-taylor-seamans-hpe-com&mode=edit"
           docs="https://v2.grommet.io/textinput?theme=hpe#props"
-          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/input/TextInputExample.js"
+          code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/components/textinput/TextInputExample.js"
+          figma="https://www.figma.com/file/E40Oo2MpdRNMyVaJqfJ0Wd/HPE-Text-Input-Component"
         >
           <TextInputExample />
         </Example>
       </Subsection>
     </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          The TextInput component allows the user to input shorter forms of data
+          and content. Passwords and tags can also be used with the TextInput
+          component. Style can be variable, based upon the use case and customer
+          need that will elicit user confidence in success.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="About TextInput" level={3} gap="small">
+        <SubsectionText>
+          Text input fields perform text validation. Some use cases for
+          TextInput include username fields, password fields, and search fields.
+        </SubsectionText>
+        <SubsectionText>
+          A TextInput display the following states: enabled, focused, focused
+          with value, validation, and disabled.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="Accessibility" level={3} gap="small">
+        <SubsectionText>
+          In every case possible, TextInput should be used inside of a FormField
+          to ensure that a label is appropriately paired with the input. This
+          behavior is important to screen reader users who need to know what
+          context the TextInput is referring to.
```

Comment:
Trying to avoid ending sentence in preposition... 

```suggestion
          behavior is important to screen reader users who need to know to
          which context the TextInput is referring.
```